docs/zookeeper.md

```diff
@@ -0,0 +1,54 @@
+# `Zookeeper` Metrics Receiver
+
+The Zookeeper receiver collects metrics from a Zookeeper instance, using the `mntr` command. The `mntr` 4 letter word command needs
+to be enabled for the receiver to be able to collect metrics.
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your Apache web server configuration.
+
+To configure a receiver for your Zookeeper metrics, specify the following fields:
+
+| Field                 | Default                 | Description                                                                            |
+|-----------------------|-------------------------|----------------------------------------------------------------------------------------|
+| `type`                | required                | Must be `zookeeper`.                                                                   |
+| `endpoint`            | `localhost:2181`        | The url exposed by zookeeper                                                           |
+| `collection_interval` | `60s`                   | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example configuration.
+
+```yaml
+metrics:
+  receivers:
+    zookeeper:
+      type: zookeeper
+      endpoint: localhost:2181
```

Comment:
Same reason as https://github.com/GoogleCloudPlatform/ops-agent/pull/384#issue-1123802764

Can we remove `endpoint` and `collection_interval` from the sample config?

Reply:
Looks like this one has not yet been changed (This change does not require re-running the integration tests).